Core/Core/Inbox/ViewModel/InboxViewModel.swift

```diff
@@ -47,15 +48,22 @@ public class InboxViewModel: ObservableObject {
     public let courseDidChange = CurrentValueSubject<InboxCourse?, Never>(nil)
     public let updateState = PassthroughSubject<(messageId: String, state: ConversationWorkflowState), Never>()
     public let contentDidScrollToBottom = PassthroughSubject<Void, Never>()
-
+    public let starDidTap = PassthroughSubject<(Bool, String), Never>()
     // MARK: - Private State
     private static let DefaultScope: InboxMessageScope = .inbox
     private let interactor: InboxMessageInteractor
+    private let messageInteractor: InboxMessageFavouriteInteractor
```

Comment:
Could we please make the interactor namings consistent? Something like
```
    private let messageInteractor: InboxMessageInteractor
    private let favouriteInteractor: InboxMessageFavouriteInteractor
```
Please update the init parameters and InboxAssembly accordingly.